source/_patterns/01-molecules/components/compact-form.yaml

```diff
@@ -17,13 +17,24 @@ schema:
       enum:
         - GET
         - POST
-    state:
+    variant:
       type: string
       enum:
         - valid
         - error
+        - info
```

Comment:
I think we should lose this, and leave `state`. (It's always one of valid, invalid or not yet completed.)